javascript/adonisjs-slim/config.yaml

```
@@ -0,0 +1,3 @@
+framework:
+  website: adonisjs.com
+  version: 5.9.0
```

```suggestion
  version: 5.9
```

using `X.Y` is more understandable